front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Toolbox/index.tsx

```diff
@@ -103,6 +105,18 @@ const HomepageBuilderToolbox = ({
             label={formatMessage(openToParticipationTitle)}
           />
         )}
+        {newHomepageWidgetsEnabled && (
```

Comment:
I prefer `new`, because we will remove this feature flag from the codebase when we release the full feature. Just `homepageWidgetsEnabled` is also not correct because we already have other homepage widgets which are not controlled by this feature flag

Reply:
Ah gotcha! Wasn't aware we'd be removing this feature flag :+1: Clear for me. Thanks!